test/lazy.test.js

```diff
@@ -57,6 +57,11 @@ describe('lazy', () => {
 		);
 		await new Promise(r => setTimeout(r, 1))
 
-		expect(ref.current.constructor).to.equal(A);
+		if (ref.current instanceof A) {
+			expect(ref.current.constructor).to.equal(A);
+		} else {
+			expect(ref.current).to.equal(scratch.firstChild);
+		}
```

Comment:
Mind if we add a couple comments to clarify the branch?

```suggestion
		if (ref.current instanceof A) {
			// v10
			expect(ref.current.constructor).to.equal(A);
		} else {
			// v11+
			expect(ref.current).to.equal(scratch.firstChild);
		}
```